cid/helpers/cur_proxy.py

```diff
@@ -0,0 +1,452 @@
+import re
+import logging
+
+logger = logging.getLogger(__name__)
+
+cur2to1mapping = {
```

Comment:
Are we supposed to create 3 more of this for each scenario?
CUR1 -> CUR1_proxy 
CUR1 -> CUR2_proxy 
CUR2 -> CUR2_proxy

Reply:
Trying to avoid this and reuse to maximum this mapping